src/concrete/ml/quantization/post_training.py

```diff
@@ -710,6 +714,11 @@ def quantize_module(self, *calibration_data: numpy.ndarray) -> QuantizedModule:
             onnx_model=self.numpy_model.onnx_model,
         )
 
+        # Set the preprocessing onnx in the quantized module
+        quantized_module._onnx_preprocessing = (  # pylint: disable=protected-access
```

Comment:
what is going on here exactly ?

Reply:
Once the quantized module is created, we need to pass the pre-processing from the numpy_module to the quantized_module. You want a setter method?

Reply:
imo it would be better to pass it to the QM's init then, especially since we already do ` onnx_model=self.numpy_model.onnx_model` 

side note: would have been much clearer to instead have a `from_numpy_module` method for QMs, or just simply only give a `numpy_module` + `quant_layers_dict` in the init, but that's not urgent

Reply:
Done.